Post about JavaFX Day (#103)
karakun/karakun.github.io: _posts/2018-11-04-jfx-day.md

---
layout: post
title:  'Karakun at JavaFX Days 2018'
author: hendrik
featuredImage: jfx-day
excerpt: 'The first ever “JavaFX-Only” conference will take place from December 3rd to 5th, 2018. The ambitious schedule 

> of the JavaFX Days includes trainings, presentations, demos and a “Java on Mobile” day. The organizer of this 3 day event

Reviewer: ```suggestion
of the JavaFX Days includes trainings, presentations, demos and a “Java on Mobile” day. The organiser of this 3 day event
```

was able to recruit some of the best and most experienced JavaFX developers for the trainings and presentations. Karakun 
is proud sponsor of this extraordinary event. '
categories: [Conferences, JavaFX]
header:
  image: post
---

> The Renaissance Tower Hotel in Zurich will be the host of the first ever “JavaFX-Only” event from December 3rd to 5th, 2018. The ambitious schedule of the JavaFX Days includes training sessions on day 1, a full day of presentations and demos on day 2, and a “Java on Mobile” day 3. DLSC Software & Consulting, the organizer of this 3 day event,  was able to recruit some of the best and most experienced JavaFX developers for the trainings and presentations. Karakun is proud sponsor of this extraordinary event. 

Reviewer: ```suggestion
The Renaissance Tower Hotel in Zurich will be the host of the first ever “JavaFX-Only” event from December 3rd to 5th, 2018. The ambitious schedule of the JavaFX Days includes training sessions on day 1, a full day of presentations and demos on day 2, and a “Java on Mobile” day 3. DLSC Software & Consulting, the organiser of this 3 day event,  was able to recruit some of the best and most experienced JavaFX developers for the trainings and presentations. Karakun is proud sponsor of this extraordinary event. 
```

![Logo]({{ "/assets/posts/2018-11-04-jfx-day/logo.png" | absolute_url }})

## Day 1: Training
Kickstart your JavaFX development efforts and get coached by two of the greatest JavaFX experts currently known: Anton Epple and [Hendrik Ebbers](/people/hendrik.html). The expert track with Hendrik Ebbers covers the following topics:

> - Scene Builder and FXML for Advanced Users

Reviewer: ```suggestion
- Scene Builder and FXML for advanced users
```

> - Transformation and Component Hierarchy in Scene Graph

Reviewer: ```suggestion
- Transformation and component hierarchy in Scene Graph
```

> - Layout Cycle and Custom Layout Containers

Reviewer: ```suggestion
- Layout cycle and custom layout containers
```

> - Advanced Custom Controls​

Reviewer: ```suggestion
- Advanced custom controls​
```

> - ​Working with Effects

Reviewer: ```suggestion
- ​Working with effects
```

> - Animation Support in JavaFX

Reviewer: ```suggestion
- Animation support in JavaFX
```

> - CSS Expertise

Reviewer: ```suggestion
- CSS expertise
```

> - Styleable Properties

Reviewer: ```suggestion
- Styleable properties
```

- Canvas
- 3D

## Day 2: Presentations

> "UI Makeover", "Tips & Tricks", "Deploy to Mobile", and so on - enjoy a variety of talks covering all kinds of aspects of JavaFX. Karakun experts, Java Champions, Java Rockstars and book authors [Hendrik Ebbers](/people/hendrik.html) and [Gerrit Grunwald](/people/gerrit.html) will talk about the following topics. Visit [the conference page](https://www.javafx-days.com/sessions) for the full list of talks.

Reviewer: ```suggestion
"UI Makeover", "Tips & Tricks", "Deploy to Mobile", and more - enjoy a variety of talks covering all kinds of aspects of JavaFX. Karakun experts, Java Champions, Java Rockstars and book authors [Hendrik Ebbers](/people/hendrik.html) as well as [Gerrit Grunwald](/people/gerrit.html) will talk about eye candy or custom controls. Visit [the conference page](https://www.javafx-days.com/sessions) for the full list of talks.
```
